### ISSUE TITLE
jmv suggest unexpected behavior

### ISSUE THREAD

**Issue by karchjd** [jamovi/jmvtools#9]
I put jmv in the "Suggests" field because I need it for my testthat tests. That led to it being included in the .jmo file. After removing it from the "Suggests" field, it was still included in the .jmo file. I had to remove the build folder to get rid of it. Is this intended?

My expectation was that suggest is not installed, like in R packages, and, even if that is the case, it's not included anymore, when removed from "Suggests" .

**Comment by jonathon-love** [comment]
you're quite correct. not intended ... if you want to remove a dependency from your module, you do need to nuke your build folder as you discovered.
